tests/dummy/app/models/dummy.js

```diff
@@ -1,40 +1,38 @@
-import Ember from 'ember';
+import EmberObject from '@ember/object';
 import { validator, buildValidations } from 'ember-cp-validations';
 
 var Validations = buildValidations({
   username: validator('length', {
     message: 'oops, {description} length is invalid',
     descriptionKey: 'errors.usernameDescription',
     min: 4,
-    max: 8
+    max: 8,
   }),
   password: [
     validator('presence', true),
     validator('length', {
       message: 'oops, length is invalid',
       min: 4,
-      max: 8
-    })
+      max: 8,
+    }),
   ],
   passwordConfirmation: validator('confirmation', {
     on: 'password',
-    messageKey: 'errors.passwordConfirmation'
+    messageKey: 'errors.passwordConfirmation',
   }),
-  email: [
-    validator('presence', true),
-    validator('format', { type: 'email' })
-  ],
+  email: [validator('presence', true), validator('format', { type: 'email' })],
   emailConfirmation: [
     validator('presence', true),
     validator('confirmation', {
       on: 'email',
-      description: 'Email addresses'
-    })
-  ]
+      description: 'Email addresses',
+    }),
+  ],
 });
 
-export default Ember.Object.extend(Validations, {
+// eslint-disable-next-line ember/no-classic-classes
+export default EmberObject.extend(Validations, {
   username: '',
   password: '',
-  email: ''
+  email: '',
 });
```

Comment:
You can use native syntax here:

```js
export default class extends EmberObject.extend(Validations) {
  username = '';
  ...
}
```

Reply:
https://github.com/ember-intl/cp-validations/pull/90/commits/131657a7eb4e28266461c4e674ffa91ac94a8380